src/components/Message/MessageOptions.tsx

```diff
@@ -57,13 +57,20 @@ const UnMemoizedMessageOptions = <
     threadList,
   } = useMessageContext<StreamChatGenerics>('MessageOptions');
 
+  const { CustomMessageActionsList } = useComponentContext('MessageOptions');
+
   const { t } = useTranslationContext('MessageOptions');
 
   const handleOpenThread = propHandleOpenThread || contextHandleOpenThread;
 
   const messageActions = getMessageActions();
-  const showActionsBox =
-    showMessageActionsBox(messageActions, threadList) || !!customMessageActions;
+  const renderMessageActions = shouldRenderMessageActions({
+    // @ts-ignore
```

Comment:
Why do we have this here?

Reply:
Unused generics, will clean up, converting PR to draft for now.